app/build.gradle.kts

```diff
@@ -56,6 +56,9 @@ android {
     abortOnError = false
     disable.addAll(arrayOf("VectorPath", "NestedWeights", "ContentDescription", "SmallSp"))
   }
+  buildFeatures {
+    viewBinding = true
+  }

```

Comment:
`buildFeatures.viewBinding` is [enabled for all Android modules](https://github.com/AndroidIDEOfficial/AndroidIDE/blob/3010fefb5a7f6c074b742340cb539cdd3e4097ba/build-logic/ide/src/main/java/com/itsaky/androidide/plugins/conf/AndroidModuleConf.kt#L109). This is unnecessary.